docs/state-committees/quickstart.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Lets make quickstart the first page. Please remove "Overview" section

Reply:
The Overview section sort of provides that introduction to what the protocol is all about. It is very good for media outlet looking for quick accurate info on the LSC @kashishshah

Reply:
I was thinking we retain the overview section particularly for media outlet or new potential users looking for a quick overview of what the product in the protocol is all about.

Reply:
agreed